.github/workflows/push.yaml

```diff
@@ -0,0 +1,50 @@
+name: push
+on: push
+permissions:
+    contents: read
+    id-token: write
```

Comment:
Do we need an ID token for this? I don't think we're using it in this.

Reply:
thanks, removed it 👍